config/schedule.rb

```diff
@@ -0,0 +1,5 @@
+require File.expand_path(File.join(File.dirname(__FILE__), "..", "config", "environment"))
+
+every 10.minutes do
+  runner "ApplicationSynchronizationJob.perform_now", environment: Rails.env
```

Comment:
Could we do `perform_later` here instead? I think it would be better for the job to be performed by the worker rather than ctron process (I am not sure how the logs would work for cron worker)

Reply:
That makes sense!

Reply:
Yea, will definitely be changing it back to `perform_later` . Once everything's tested and sorted